source/appModules/devenv.py

```diff
@@ -473,3 +486,36 @@ class IVsTextManager(comtypes.IUnknown):
     COMMETHOD([], HRESULT, 'SetTopLine',
               ( ['in'], c_int, 'iBaseLine' )),
 ]
+
+class UIATextInfoForVisualStudio( UIATextInfo):
+
+	def __init__(self,obj,position,_rangeObj=None):
+		super(UIATextInfoForVisualStudio,self).__init__(obj,position,_rangeObj)
+		self.digitsAtStartOfLineMatch = re.compile(r"^[\d]+\s")

```

Comment:
nit: You could just define this as a class-level constant; i.e. you don't need a constructor for this.